docs/manage/_content.rst

```diff
@@ -178,6 +184,30 @@ If Kolibri is installed on an older or a low-resource device, you can try the fo
 On Linux and OSX you will need to enable the **Show hidden folders** option in order to view the ``.kolibri`` folder.    
 
 
+Update Content Channels
+-----------------------
+
+Public channels on Kolibri Studio are periodically updated from their original sources. Private and unlisted channels that you have on you local Kolibri device may have also changed since you first imported them, and have some new content added in the meantime. You can update your local channels from Kolibri Studio if you have internet connection, from another device in your local network where the content has already been updated, or from an external drive where the latest version of the channels has been :ref:`previously exported <export>`.
+
+.. warning:: Don't update channels used to make :ref:`lessons <manage_lessons>` and :ref:`quizzes <manage_quizzes>` while those are **still active and visible to learners**. The structure and location of some resources inside the channel may have changed in the updated version, which can render them unavailable and create loading issues. It is recommended to **schedule and perform channel updates only during periods when learners are not actively working on lessons and quizzes**.
+
+To update channel to its latest version and import new content into Kolibri, follow these steps.
+
+#. Click :guilabel:`OPTIONS` for the channel you want to update, and select **Import more**.
+#. Choose the source option: **Kolibri Studio**, **Local network or internet**, or **Attached drive or memory card**.
+#. If there is a new channel version available, you will see the notification.
+
+    .. figure:: img/update-channel.png
+      :alt: 
+
+      Notification for the new version of the content channel.
+
+4. Click the :guilabel:`UPDATE` button, and wait for the updated topic tree to load.
+5. You will see empty checkboxes for any new content added to the channel since you last imported. Select and import content by following the steps for import from :ref:`Kolibri Studio <import_studio>`, :ref:`local network <local_network>`, or :ref:`attached drive <local_drive>`.
```

Comment:
```suggestion
5. You will see empty check boxes for any new content added to the channel since your last import. Select and import content by following the steps from :ref:`Kolibri Studio <import_studio>`, :ref:`local network <local_network>`, or :ref:`attached drive <local_drive>`.
```